mattermost-plugin/webapp/tests/utils.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import {IntlProvider} from 'react-intl'
+
+export const wrapIntl = (children?: React.ReactNode): JSX.Element => (
+    <IntlProvider locale='en'>
+        {children}
+    </IntlProvider>
+)
```

Comment:
We already have [wrapIntl](https://github.com/mattermost/focalboard/blob/main/webapp/src/testUtils.tsx#L12)